### DISCUSSION TITLE
using $app.dao() with node

### DISCUSSION THREAD

**Discussion by rogerpence** [pocketbase/pocketbase#5056] [Q&A]
How do you get a reference to the $app object with Node. The docs say that the $app object is one of many objects "that are accessible from everywhere" on [this page](https://pocketbase.io/docs/js-overview/#global-objects).

**Comment by ganigeorgiev** [comment]
The [JSVM](https://pocketbase.io/docs/js-overview/) is not a Node environment. Everything in the "Extend with JavaScript" section is only available within `pb_hooks` and `pb_migrations`.
